src/lib/SCREEN/OLED/oleddisplay.cpp

```diff
@@ -396,6 +397,48 @@ void OLEDDisplay::displaySending()
     u8g2->sendBuffer();
 }
 
+void OLEDDisplay::displayLinkstats()
+{
+    u8g2->clearBuffer();
+    u8g2->setFont(u8g2_font_profont10_mr);
+
+    u8g2->drawStr(0, 20, "LQ");
+    u8g2->drawStr(0, 30, "RSSI");
+    u8g2->drawStr(0, 40, "SNR");
+    u8g2->drawStr(0, 50, "Ant");
+
+    u8g2->drawStr(32, 10, "Uplink");
+    u8g2->setCursor(32, 20);
+    u8g2->print(CRSF::LinkStatistics.uplink_Link_quality);
+    u8g2->setCursor(32, 30);
+    u8g2->print((int8_t)CRSF::LinkStatistics.uplink_RSSI_1);
+    u8g2->print("/");
```

Comment:
Maybe wrap RSSI_2 in something that only prints if it is non-zero?

I've also just realized if we switch from a diversity RX to a non-diversity RX, our linkstats will keep displaying RSSI_2 from the previous session. I am not sure how the heck we'd fix that though without zeroing out the values on disconnect which could be throwing away important information at range.